### PR TITLE
Print only new packages added

### PR DESCRIPTION
**Pull request by colombod**

### CODE REVIEW

**Comment by KevinRansom** [comment]
They are indeed very invasive, I wonder if we can add some form diagnostic mode.

**Review by KevinRansom — APPROVED** [review]

File: src/Microsoft.DotNet.Interactive.NetStandard20/Microsoft.DotNet.Interactive.Netstandard20.csproj [hunk 1]
```diff
@@ -11,6 +11,8 @@
 
   <ItemGroup>
     <Compile Include="$(MSBuildThisFileDirectory)../Microsoft.DotNet.Interactive/**/*.cs" />
+    <Compile Remove="$(MSBuildThisFileDirectory)../Microsoft.DotNet.Interactive/bin/**/*.cs" />
```
Comment (KevinRansom) [inline src/Microsoft.DotNet.Interactive.NetStandard20/Microsoft.DotNet.Interactive.Netstandard20.csproj:14]: Why is this necessary, shouldn't generated files be included?

Comment (colombod) [reply src/Microsoft.DotNet.Interactive.NetStandard20/Microsoft.DotNet.Interactive.Netstandard20.csproj:14]: @brettfo found that they get included twice, running a dotnet build is causing that issue

Comment (brettfo) [reply src/Microsoft.DotNet.Interactive.NetStandard20/Microsoft.DotNet.Interactive.Netstandard20.csproj:14]: In Diego's environment, the auto-generated files are under `bin/` and `obj/` and the netstandard2.0 project should certainly include its own generated files, but not those from the non-netstandard2.0 project.

**Review by brettfo — APPROVED** [review]
